test/unit/processing/siti-agri/handle-known-defects/cap-bps-penalties.test.js

```diff
@@ -0,0 +1,43 @@
+const { capBPSPenalties } = require('../../../../../app/processing/siti-agri/handle-known-defects/cap-bps-penalties')
+const { bps, sfi } = require('../../../../../app/schemes')
+const { P02, P04 } = require('../../../../../app/constants/line-descriptions')
```

Comment:
I think the order of these should be { P02, P04 }, { bps, sfi } and then { capBPSPenalties } with a space between { P02, P04 } and { bps, sfi }/{ capBPSPenalties }